hack/cherry_pick_pull.sh

```diff
@@ -182,4 +186,10 @@ if ! [[ "${REPLY}" =~ ^[yY]$ ]]; then
 fi
 
 git push origin -f "${NEWBRANCHUNIQ}:${NEWBRANCH}"
+
+if [[ -n "${DRY_RUN}" ]]; then
```

Comment:
Actually, it does leave you in a different branch than you were in.  I'll make a note of that when I update.